content/docs/user-guide/experiment-management/experiments-overview.md

```diff
@@ -0,0 +1,67 @@
+# DVC Experiments Overview
+
+DVC Experiments are captures automatically by DVC when you [run them]. Each
+experiment creates and tracks a project variation based on the changes in your
+<abbr>workspace</abbr>. Experiments preserve the latest commit in the current
+branch (Git `HEAD`) as their parent or _baseline_, but do not form part of the
+regular Git tree or workflow (unless you make them [persistent]).
+
+<details>
+
+### ⚙️ How does DVC track experiments?
+
+Experiments are custom [Git references] (found in `.git/refs/exps`) with a
+single commit based on `HEAD` (not checked out by DVC). Note that these commits
+are not pushed to Git remotes by default (see `dvc exp push`).
+
+</details>
+
+[run them]: /doc/user-guide/experiment-management/running-experiments
+[git references]: https://git-scm.com/book/en/v2/Git-Internals-Git-References
+
+## Properties
+
+DVC Experiments will have an auto-generated name like `exp-bfe64` by default. A
+custom name can be given instead (using the `--name`/`-n` option of
+`dvc exp run`). These names can be used to reference experiments in other
+`dvc exp` subcommands.
+
+All experiments created by DVC will be associated to the latest commit (Git
+`HEAD`) at the time that they were run. This is called the experiment's
+_baseline_.
+
+## Basic workflow
+
+`dvc exp` commands let you automatically track a variation to a committed
+project version (baseline). You can create independent groups of experiments
+this way, as well as review, compare, and restore them later. The basic workflow
+goes like this:
+
+- Modify hyperparameters or other dependencies (input data, source code,
+  commands to execute, etc.). Leave these changes un-committed in Git.
```

Comment:
It might help to discuss why we have this workflow and want to leave changes un-committed. In other experiment tracking tools, the workflow looks like:

1. Run the experiment.
2. Log the commit hash of the current HEAD, even though it doesn't include your dirty changes.
3. If everything is successful, save changes in a new commit.

This creates a confusing state where the experiment should really be associated with the second commit instead of the first.

It might be too much detail or inappropriate for the page, but maybe it can be summarized, or it might spark other ideas. This is probably a good point for the blog post but not sure if there's a place for it...

Reply:
> Similar to the discussion up in https://github.com/iterative/dvc.org/pull/2909#pullrequestreview-812329316

Yeah this needs more work. I'm hoping for now it's mergeable but I'll improve is as much as possible ⌛

Reply:
> good point for the blog post

Do you mean linking from here? It's probably more important to link from the blog post to our docs, I think.

Reply:
> Do you mean linking from here?

No, just thinking out loud that this idea would be good to incorporate into the blog post.